markup/asciidoc/asciidocext_config/config.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Hugo Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package asciidoc_config holds asciidoc related configuration.
+package asciidocext_config
+
+// DefaultConfig holds the default asciidoc configuration.
+var (
+	Default = Config{
+		Backend:          "html5",
+		DocType:          "article",
+		Extensions:       []string{""},
+		NoHeaderOrFooter: true,
+		SafeMode:         "safe",
+		SectionNumbers:   false,
+		Verbose:          false,
+	}
+
+	ExtensionsWhitelist = []string{
```

Comment:
I think it's easier to use a map[string]bool for these whitelists. Also, no need to export them (use lower case first letter).

Reply:
@bep I change it to `map[string]bool`, but I had to export the variables because the `asciidoc` module uses them in the `parseArgs()` method.

The reason `parseArgs` is in `convert.go` is because attempting to pass a `converter.DocumentContext` object would cause a cyclic dependency issue. @muenchhausen 's fork uses that `ctx`.